test/test-pages/wordpress/expected-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Stack Overflow Jobs Data Shows ReactJS Skills in High Demand, WordPress Market Oversaturated with Developers – WordPress Tavern",
```

Comment:
I thought we'd be trying to get rid of the "WordPress Tavern" bit, too. Why isn't that working? Maybe that wants to be another issue?